### PR TITLE
Togglebuttons

### LINKED ISSUE #102: Images to buttons on spawning panels
Improve spawning panel design according to discussions. Add images instead to buttons/checkboxes.

### PR DESCRIPTION
**Pull request by DecibelM**
Closes #102 

*Changed togglebuttons to buttons with images
*Some general fixup of the position of things on spawning panels.

### CODE REVIEW

File: KA-Flocking/Assets/Scripts/ToggleSwitch.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    public GameObject ArcherToggleOff;
+    public GameObject ScoutToggleOn;
+    public GameObject ScoutToggleOff;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/ToggleSwitch.cs]: Remove unused function

File: KA-Flocking/Assets/Scripts/ToggleSwitch.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        if (CircularToggle.isOn)
+        {
+            CircularToggleOn.SetActive(true);
+            CircularToggleOff.SetActive(false);
+        }
+        else if (!CircularToggle.isOn)
+        {
+            CircularToggleOn.SetActive(false);
+            CircularToggleOff.SetActive(true);
+        }
```
Comment (Hoidi) [inline KA-Flocking/Assets/Scripts/ToggleSwitch.cs]: Couldn't this be 
`CirculatToggleOn.SetActive(CircularToggle.isOn);`
`CirculatToggleOff.SetActive(!CircularToggle.isOn);`

Comment (DecibelM) [reply KA-Flocking/Assets/Scripts/ToggleSwitch.cs]: Ah smart, I'll check it out!

**Comment by DecibelM** [comment]
These things fixed:
*Anchoring for panels
*Switched image for pike
*Removed unused method
*More efficient code (THANK YOU!!)
:)

**Pull request merged**